governance/votes/Executive vote - April 15, 2022.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Template - [Executive Vote] Offload TUSD, Curve ETH-stETH Vault Parameter Changes, DAI Transfers - April 15, 2022
+summary: Offload TUSD collateral via liquidation, increase CRVV1ETHSTETH-A Maximum Debt Ceiling and Target Available Debt, DAI Transfers to GRO-001, Ambassadors Program, Gelato Keepers.
```

Comment:
Should Ambassador be singular or plural? Kat used both in her forum thread. Personally I feel Ambassador Program scans better